c_src/quicer_listener.c

```diff
@@ -47,6 +51,37 @@ ServerListenerCallback(__unused_parm__ HQUIC Listener,
 
       c_ctx->Connection = Event->NEW_CONNECTION.Connection;
 
+      /* reload trusted store very time to make sure we incorporate
+       * any changes to the file
+       */
+      if (l_ctx->cacertfile)
+        {
+          X509_STORE *trusted = NULL;
+          X509_LOOKUP *lookup = NULL;
+          trusted = X509_STORE_new();
+
+          if (trusted != NULL)
+            {
+              lookup = X509_STORE_add_lookup(trusted, X509_LOOKUP_file());
+              if (lookup != NULL)
+                {
+                  if (!X509_LOOKUP_load_file(lookup,
```

Comment:
This will be called for every new connection,  and it is costly to call and will soon become the bottleneck

It will get improved when we get this merged to msquic.
https://github.com/microsoft/msquic/pull/3132/files.

Reply:
Yes, I agree. That should be solved if/when my PR to msquic is accepted.

Reply:
Then the cacertfile is passed on to msquic and incorporated into the SSL ctx from the start.

Reply:
In the meanwhile, one alternative would be to allocate the trusted structure earlier and re-use that every time instead. The only drawback of that is that it will not be updated if the contents of the cacertfile is changed.

Reply:
Let me know what you want me to do. Wait for the msquic change or optimize the initializtion of `trusted`.

Reply:
> The only drawback of that is that it will not be updated if the contents of the cacertfile is changed.
Agree, in that case We need some API to update/revoke the cacert.